module/actor.js

```diff
@@ -362,6 +362,11 @@ class DCCActor extends Actor {
       }
     ]
 
+    // Initiative: A warrior add his class level to his initiative rolls.
+    if ((token._actor.system.class.className == "Warrior") && (game.settings.get('dcc', 'automateWarriorInititative'))) {
+      terms[1].formula = terms[1].formula + " + " + token._actor.system.details.level.value;
```

Comment:
Can this be added on as a term like in the change to attach rolls, for compatibility with the roll modifer dialog?

Players might often want to modify this, but the modifier dialog is also useful to see where elements of a roll are coming from.

Reply:
Added as a separate term.